solidity/test/TestPhasedEscrow.js

```diff
@@ -476,6 +509,69 @@ describe("StakerRewardsBeneficiary", () => {
     })
   })
 })
+
+describe("BeaconRewards to PhasedEscrow transfer", async () => {
+  const owner = accounts[0]
```

Comment:
We don't need to redefine `owner` as it's already done in L36.

Reply:
It's another `describe`, owner from L36 is not available here. I wanted to keep it separately as we need to declare all contracts from scratch to perform this test (`initRewards`).

Reply:
What also made me realize I missed declarations for some variables. Fixed in https://github.com/keep-network/keep-core/pull/2159/commits/058e0ca390eb4f4496395081d54516a8b54bf082.